### PR TITLE
Temporarily disable MNIST tests

### PR DESCRIPTION
**Pull request by malfet**

### CODE REVIEW

File: .jenkins/pytorch/test.sh [hunk 1]
```diff
@@ -194,8 +194,9 @@ test_libtorch() {
       build/bin/test_jit "[cpu]"
     fi
     python test/cpp/jit/tests_setup.py shutdown
-    python tools/download_mnist.py --quiet -d test/cpp/api/mnist
-    OMP_NUM_THREADS=2 TORCH_CPP_TEST_MNIST_PATH="test/cpp/api/mnist" build/bin/test_api
+    # Temporarily disable MNIST tests
+    # python tools/download_mnist.py --quiet -d test/cpp/api/mnist
+    OMP_NUM_THREADS=2 TORCH_CPP_TEST_MNIST_PATH="test/cpp/api/mnist" build/bin/test_api --gtest_filter='-*MNIST*'
```
Comment (mrshenli) [inline .jenkins/pytorch/test.sh]: Do we need to comment out the following in test_xla?

```
  echo "Running MNIST Test"
  python test/test_train_mnist.py --tidy
```

**Comment by ezyang** [comment]
I'm pretty sure we also exercise MNIST from Python.

**Review by ezyang — APPROVED** [review]
pre-approving but I don't think is gonna be enough

**Review by mrshenli — COMMENTED** [review]
MNIST s also used in `test/onnx/test_models.py` and `test/onnx/test_pytorch_onnx_caffe2.py`. Do we need to disable them as well?

**Comment by malfet** [comment]
> MNIST s also used in `test/onnx/test_models.py` and `test/onnx/test_pytorch_onnx_caffe2.py`. Do we need to disable them as well?
@mrshenli `download_mnist` is only references from the files I've changed, and `test_models.py` and `text_pytorch_onnx_caffe2.py` seems to be downloading their data from another location

**Review by seemethere — CHANGES_REQUESTED** [review]
I believe this should be a sufficient alternative: https://github.com/pytorch/pytorch/pull/34222

**Comment by malfet** [comment]
Agreed.

**Pull request closed without merging**